tests/scripts/generate_config_tests.py

```diff
@@ -105,60 +111,60 @@ def find_super_option(cfg: config.Config,
         return m.group('prefix') + 'BASIC'
     return None
 
-def conditions_for_option(cfg: config.Config,
-                          setting: config.Setting
-                          ) -> Iterator[Tuple[List[str], str]]:
+def conditions_for_setting(cfg: config.Config,
+                           setting: config.Setting
+                           ) -> Iterator[Tuple[List[str], str]]:
     """Enumerate the conditions under which to test the given setting.
 
-    * cfg: all configuration options.
+    * cfg: all configuration settings.
     * setting: the setting to be tested.
 
     Generate a stream of conditions, i.e. extra dependencies to test with
     together with a human-readable explanation of each dependency. Some
     typical cases:
 
     * By default, generate a one-element stream with no extra dependencies.
-    * If the setting is ignored unless some other option is enabled, generate
-      a one-element stream with that other option as an extra dependency.
-    * If the setting is known to interact with some other option, generate
-      a stream with one element where this option is on and one where it's off.
+    * If the setting is ignored unless some other setting is enabled, generate
+      a one-element stream with that other setting as an extra dependency.
+    * If the setting is known to interact with some other setting, generate
+      a stream with one element where this setting is on and one where it's off.
     * To skip the setting altogether, generate an empty stream.
     """
     name = setting.name
     if name.endswith('_ALT') and not config.is_seamless_alt(name):
         # We don't test alt implementations, except (most) platform alts
         return
-    super_setting = find_super_option(cfg, setting)
+    super_setting = dependencies_of_setting(cfg, setting)
```

Comment:
```suggestion
    dependencies = dependencies_of_setting(cfg, setting)
```

Reply:
I amended the last commit to take care of this.